testsuite/Rakefile

```diff
@@ -34,22 +38,25 @@ namespace :parallel do
   Dir.glob(File.join(Dir.pwd, 'run_sets', '*.yml')).each do |entry|
     run_set = File.basename(entry, '.yml').to_sym
     desc "Run Cucumber #{run_set} features in parallel"
-    task "#{run_set}" do
+    task run_set.to_s do
       timestamp = Time.now.strftime('%Y%m%d%H%M%S')
+      json_results = "-f json -o output_#{timestamp}-#{run_set}-$TEST_ENV_NUMBER.json"
+      html_results = "-f html -o output_#{timestamp}-#{run_set}-$TEST_ENV_NUMBER.html"
       features = YAML.safe_load(File.read(File.join(Dir.pwd, 'run_sets', "#{run_set}.yml"))).join(' ')
       profiles = ENV['PROFILE']
-      include_profiles = profiles.nil? ? '--profile default' : '--profile ' + profiles.gsub(',', ' --profile ')
+      include_profiles = profiles.nil? ? '--profile default' : "--profile #{profiles.gsub(',', ' --profile ')}"
       tags = ENV['TAGS']
       include_tags =  tags.nil? ? '' : "--tags #{tags}"
-      cucumber_opts = "#{include_profiles} #{include_tags} -f html -o output_#{timestamp}-#{run_set}-$TEST_ENV_NUMBER.html -f json -o output_#{timestamp}-#{run_set}-$TEST_ENV_NUMBER.json #{junit_results} -f CustomFormatter::PrettyFormatter -r features "
+      cucumber_opts = "#{include_profiles} #{include_tags} #{html_results} #{json_results} #{junit_results} -f rerun " \
```

Comment:
I assumed the `-f` flag would be for file. May I ask what the `-f rerun` does exactly?

Reply:
uhh good catch, old try we don't want to have anymore.
It retries failed features.

Reply:
(-f is for format)